### DISCUSSION TITLE
Is it possible to expand support to PowKiddy V90 / Q90?

### DISCUSSION THREAD

**Discussion by SuperBadger77** [eduardofilo/RG350_adam_image#111] [Ideas & Requests]
Hi,

I've seen that retroarch now supports: Miyoo – Pocket Go, PowKiddy Q90-V90 and New BittBoy: https://www.libretro.com/index.php/retroarch-1-9-12-release/ will you be extending support to the Q90 or V90? I just checked my V90 and it only has a single SD card so not sure if that would work with the Adam Image?

**Comment by eduardofilo** [comment]
Hi.

It has a completely different architecture (ARM) so it would require making the image from scratch, and I have enough with the work that one gives. :-)
But most of all, I don't have that machine.

**Comment by SuperBadger77** [comment]
No worries, thought I'd ask - you are doing a fantastic job, I know everyone who uses these images love them. Thank you for your service.

**Comment by eduardofilo** [comment]
Thanks. Although all this software is quite unstable and I get all kinds of problems that have to be managed. That is what I meant by the fact that the image is a lot of work and that is why I would not want to start a new one.
